minimal-notebook/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     texlive-latex-extra \
     texlive-xetex \
     unzip \
-    vim \
+    nano \
```

Comment:
`nano` only needs to be added to this one Dockerfile. All of the other images here inherit from this image and will have it installed. So it's OK to remove all the other changes besides this one line.